### ISSUE TITLE
[Bug Report][2.6.9] Hidden inputs break `v-dialog` focus trap

### ISSUE THREAD

**Issue by shengslogar** [vuetifyjs/vuetify#15745]
### Environment
**Vuetify Version:** 2.6.9
**Vue Version:** 2.7.10
**Browsers:** Chrome 105.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Place any `<input type="hidden">` inside a `<v-dialog>`.

### Expected Behavior
Focus is trapped within dialog.

### Actual Behavior
Focus trap breaks and background elements are focusable.

### Reproduction Link
<a href="https://codepen.io/shengslogar/pen/oNdjWNo" target="_blank">https://codepen.io/shengslogar/pen/oNdjWNo</a>

### Other comments
This is the result of the query selector here:

https://github.com/vuetifyjs/vuetify/blob/ac45c98983717d5ee42c4e85de1deccbd698cc83/packages/vuetify/src/components/VDialog/VDialog.ts#L236-L239

At a bare minimum, hidden inputs should be excluded from this selector. One might also consider `aria-hidden` inputs or `display: none` inputs (but the latter would require computing element dimensions which could result in a breaking change).

**Comment by shengslogar** [comment]
Additionally, the `:not([tabindex="-1"])` selector should wrap all the other selectors as I would expect the following workaround to be supported:

```
<input type="hidden" tabindex="-1">
```

**Comment by shengslogar** [comment]
Sweet, thanks @KaelWD!
